packages/web-client/app/utils/token.ts

```diff
@@ -153,8 +153,20 @@ export class TokenDisplayInfo<T extends TokenSymbol> implements DisplayInfo {
     this.icon = displayInfo.icon;
   }
 
-  static iconFor(symbol: TokenSymbol) {
-    return _tokenDisplayInfoMap[symbol].icon;
+  static iconFor(symbol: string) {
+    switch (symbol) {
+      case 'ETH':
+        return _tokenDisplayInfoMap['ETH'].icon;
+      case 'DAI':
+      case 'DAI.CPXD':
+      case 'XDAI':
```

Comment:
What token corresponds to this case? Do we accept XDAI?

Reply:
I guess we can clean this up when we start moving towards supporting more tokens? Since we should move towards a single source of truth for token display stuff by then.